app/javascript/articles/Article.jsx

```diff
@@ -3,140 +3,105 @@ import PropTypes from 'prop-types';
 import { articlePropTypes } from '../src/components/common-prop-types/article-prop-types';
 import {
   CommentsCount,
+  CommentsList,
   ContentTitle,
-  OrganizationHeadline,
-  PublishDate,
-  ReadingTime,
+  Author,
+  OverflowNavigation,
   SaveButton,
   SearchSnippet,
   TagList,
   ReactionsCount,
 } from './components';
 import { PodcastArticle } from './PodcastArticle';
 
-/* global timeAgo */
-
 export const Article = ({
   article,
   currentTag,
+  isFeatured,
   isBookmarked,
-  reactionsIcon,
-  commentsIcon,
-  videoIcon,
+  // videoIcon,
   bookmarkClick,
 }) => {
   if (article && article.type_of === 'podcast_episodes') {
     return <PodcastArticle article={article} />;
   }
 
-  const timeAgoIndicator = timeAgo({
-    oldTimeInSeconds: article.published_at_int,
-    formatter: x => x,
-  })
-
   return (
     <div
-      className="single-article single-article-small-pic"
+      className="crayons-story"
+      id={isFeatured && ("featured-story-marker")}
+      data-featued-article="TODO"
       data-content-user-id={article.user_id}
     >
-      {article.cloudinary_video_url && (
+      {isFeatured && (
```

Comment:
Soo... Instead of having two separate jsx files for featured article and normal one, i added this attribute so now we have only Article.jsx and I check with this whether or not it's featured. if yes, i show the cover and make title font bigger...

Reply:
The reason for that is these two types are almost identical so it doesn't make sense to have duplicated code :/